physx/Cargo.toml

```diff
@@ -14,13 +14,14 @@ categories = ["api-bindings", "simulation", "game-engines"]
 doctest = false
 
 [dependencies]
-physx-macros = { version = "0.1.0", path = "../physx-macros" }
-physx-sys = { version = "0.1.2", path = "../physx-sys" }
+physx-macros = { version = "0.1", path = "../physx-macros" }
+physx-sys = { version = "0.1", path = "../physx-sys" }
 
 enumflags2 = "0.5"
 enumflags2_derive = "0.5"
 log = "0.4"
-nalgebra = "0.18.0"
-nalgebra-glm = "0.4"
-ncollide3d = "0.20.1" # TODO: Remove, tracked in https://github.com/EmbarkStudios/physx-rs/issues/17
-parking_lot = "0.9.0"
\ No newline at end of file
+glam = "0.7"
+parking_lot = "0.9.0"
+
+nalgebra-glm = "*"
+nalgebra = "*"
```

Comment:
shouldn't these be gone now?

Reply:
Yes I just added some nalgebra tests. Will remove the last commit before the merge